src/concourse/lib/resource_types.py

```diff
@@ -0,0 +1,9 @@
+from concourse.lib.models import Identifier, ResourceType
+
+
+def rclone() -> ResourceType:
+    return ResourceType(
+        name=Identifier("rclone"),
+        type="docker-image",
```

Comment:
Shouldn't the `resource-image` be used instead based on [this](https://github.com/concourse/registry-image-resource)?